### ISSUE TITLE
Very low contrast ratio on certain feedback chips.

### ISSUE THREAD

**Issue by HRemonen** [UniversityOfHelsinkiCS/palaute#1234]
Atleast vastapalaute puuttuu chip does not have sufficient color contrast ratio
